doc/api/tracing.md

```diff
@@ -20,6 +20,15 @@ Running Node.js with tracing enabled will produce log files that can be opened
 in the [`chrome://tracing`](https://www.chromium.org/developers/how-tos/trace-event-profiling-tool)
 tab of Chrome.
 
+The logging file is by default called `node_trace.1.log`, where `1` is an
+incrementing log-rotation id. You can specify the filepath pattern with
+`--trace-event-file-pattern` that accepts an `printf` style format string.
```

Comment:
nit: an `printf` -> a `printf`

Reply:
Please avoid using `you` in the docs :-)